src/components/Documentation/Layout/JsonSchemas/JsonSchemasComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import s from "./JsonSchemasComponent.module.scss"
```

Comment:
can we make this descriptive instead of `s`?

Reply:
done